components/brave_rewards/browser/rewards_service_impl_unittest.cc

```diff
@@ -76,6 +90,21 @@ class RewardsServiceTest : public testing::Test {
   RewardsServiceImpl* rewards_service() { return rewards_service_; }
   MockRewardsServiceObserver* observer() { return observer_.get(); }
 
+#if BUILDFLAG(ENABLE_GEMINI_WALLET)
+  void EnableGemini() {
+    feature_list_.InitAndEnableFeature(features::kGeminiFeature);
+  }
+#endif
+
+#if BUILDFLAG(ENABLE_GEMINI_WALLET)
```

Comment:
nit: use one buildflag block for both methods

Reply:
fixed